ngrinder-frontend/package.json

```diff
@@ -74,22 +74,23 @@
     "vuex": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.2",
-    "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@babel/plugin-proposal-decorators": "^7.6.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/plugin-transform-runtime": "^7.6.2",
-    "@babel/polyfill": "^7.6.0",
-    "@babel/preset-env": "^7.6.2",
-    "@babel/plugin-transform-for-of": "^7.7.4",
+    "@babel/core": "^7.12.10",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/plugin-proposal-decorators": "^7.12.12",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.12.10",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/plugin-transform-for-of": "^7.12.1",
     "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.3",
-    "babel-loader": "^8.0.6",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
     "babel-plugin-component": "^1.1.1",
     "babel-register": "^6.26.0",
     "copy-webpack-plugin": "^5.0.4",
+    "acorn-dynamic-import": "^4.0.0",
```

Comment:
What is it for?

Reply:
@junoyoon 

When npm install, npm announce warning below.
This module is `devDependencies` so not affect builded artifact size.

`npm WARN vue-authenticate@1.4.1 requires a peer of acorn-dynamic-import@^4.0.0 but none is installed. You must install peer dependencies yourself.`

Reply:
@junoyoon 

I will check again this module is nessesary.

Reply:
@junoyoon 

I removed [vue-authenticate](https://www.npmjs.com/package/vue-authenticate) because it is not using in project.
so peer dependency with vue-authenticate named `acron-dynamic-import` is not required. I removed this too.

Reply:
Ok